Lib/test/test_ast.py

```diff
@@ -1025,6 +1026,31 @@ def test_none_checks(self) -> None:
         for node, attr, source in tests:
             self.assert_none_check(node, attr, source)
 
+    def test_repr(self):
+        const_1 = ast.Constant(1)
+        const_2 = ast.Constant(2)
+        self.assertEqual(repr(const_1), "Constant(value=1, kind=None)")
+
+        add = ast.BinOp(op=ast.Add(), left=const_1, right=const_2)
+        self.assertEqual(repr(add),
+                         "BinOp(left=Constant(value=1, kind=None), op=Add(), right=Constant(value=2, kind=None))")
+
+        self.assertEqual(repr(ast.parse("x = 3")),
+                         "Module(body=[Assign(targets=[Name(...)], value=Constant(...), type_comment=None)], "
```

Comment:
I think we should have a depth of 3 here because here this does not give you any good idea of what is being done. Here I would expect to see "x" and "3" at least, because otherwise the repr is way to generic.

Reply:
Increased the depth to 3